src/cosmosdb-preview/azext_cosmosdb_preview/vendored_sdks/azure_mgmt_cosmosdb/models/_models_py3.py

```diff
@@ -12516,6 +12516,8 @@ class RestoreParameters(RestoreParametersBase):
      list[~azure.mgmt.cosmosdb.models.GremlinDatabaseRestoreResource]
     :ivar tables_to_restore: List of specific tables available for restore.
     :vartype tables_to_restore: list[str]
+    :ivar source_backup_location: The source backup location for restore.
+    :vartype source_backup_location: str
```

Comment:
is this file autogenerated? do we need to explicitly include in this PR?

Reply:
This part of the change did not get autogenerated. I had to add this here. This is the restore parameters model, without this the test does not recognize sourceBackupLocation.